### PR TITLE
Doc reorg.

### PR DESCRIPTION
**Pull request by DavidSagan**
Reorganization of the documentation.

### CODE REVIEW

File: front_matter/doc_editing_setup.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Documentation
+# Documentation Editing Setup
+
+The documentation uses the [MyST](https://myst-parser.readthedocs.io/en/latest/) markup language.
```
Comment (EZoni) [inline front_matter/doc_editing_setup.md:3]: I wonder which one is the main website to reference, the one you have here or https://mystmd.org/? I think the one you have here might be the one for the Sphinx and Docutils extension to parse MyST, but not the one of MyST itself.

File: front_matter/introduction.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Accelerator Lattice Standard
+
+This standard is 
+an effort to create a standard to promote lattice information exchange for particle accelerators.
```
Comment (EZoni) [inline front_matter/introduction.md:4]: Could we have this sentence on one line in the source file (I think one sentence per line is standard practice to display git diffs more easily)?
```suggestion
This standard is an effort to create a standard to promote lattice information exchange for particle accelerators.
```

File: front_matter/introduction.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Scope
+
+The standard focusus on the physical layout and properties of a machine.
```
Comment (EZoni) [inline front_matter/introduction.md:8]:
```suggestion
The standard focuses on the physical layout and properties of a machine.
```

**Review by EZoni — COMMENTED** [review]
Looks good to me, I left only a few minor comments.

In general I would try to minimize duplication of content (e.g., between README and intro section) for easier maintenance, but this can be refined in future PRs as the standard makes progress and more contributions come in.

File: front_matter/doc_editing_setup.md (hunk 1 above)
Comment (DavidSagan) [reply front_matter/doc_editing_setup.md]: Changed.

File: front_matter/introduction.md (hunk 2 above)
Comment (DavidSagan) [reply front_matter/introduction.md]: Changed.

**Review by EZoni — APPROVED** [review]

**Comment by EZoni** [comment]
Thanks, @DavidSagan! I think we can merge this PR. After the merge I will open a PR to improve the guidance on how to install and run MyST.

**Pull request merged**